resources/.prettierrc

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
We use standardjs elsewhere in Brave.  Can we keep to that, to prevent one-off tooling?